src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -16,6 +16,10 @@
             <f:textbox/>
         </f:entry>
 
+        <f:entry title="${Constraints}" field="constraints" description="comma separated LTarget,RTarget,Operand">
```

Comment:
```
ltarget,operand,rtarget|ltarget,operand,rtarget
ltarget,operand,rtarget/ltarget,operand,rtarget
ltarget,operand,rtarget:ltarget,operand,rtarget
ltarget,operand,rtarget#ltarget,operand,rtarget
```

is my suggestions, not sure if they are brilliant, the best would be to have a line per constraint, can jenkins do multiple inputs ? `|`,`/`, `:`, `#`